README.md

```diff
@@ -12,6 +12,89 @@ To run docker as a non-root user, run: "sudo chmod 666 /var/run/docker.sock" (re
 To generate a new SSH key and it to the machine's SSH agent: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent?platform=linux
 Add a new SSH key to your GitHub account: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account?platform=linux
 
+## Running on a Slurm cluster
+This script must be run on a slurm cluster connected to the NFS. It will spawn all simpoints as slurm jobs to run on any node where the docker container is found. If no nodes are found it will attempt launch a container on one of the nodes using this Scarab-Infra repository (which should be on the nfs to launch containers from).
+
+### Requirements
+Must run on a slurm cluster node
+All nodes must have docker installed
+
+### Location of stats, parameters file, and scarab binary
+The stats are moved to a different location for the slurm runner. The stats for an experiment can be found in your docker home foler, under the following path:
+`/docker_home/experiment_name/configuration_name/workload/simpoint_id/`
+
+The file containing the architectural parameters and the scarab binary used by an experiment can be found at the following path:
+`/docker_home/experiment_name/`
+
+### New descriptor file options
+These options should not affect functionality of the existing infrastructure. New options are:
+
+Scarab path:
+Ex: `"scarab_path": "/path/to/scarab_bin"`
+This is a path to the scrab binary to be used. Absolute path reccomended
+
+Docker home:
+Ex: `"docker_home": "path/to/docker/container/home"`
+This is the path to the 'home' directory where all docker containers are linked. Should have been provided to build command via '-o' option
+
+Docker prefix:
+Ex: `"docker_prefix": "app_groupname"`
+This corresponds to the APP_GROUPNAME used in the existing infrastructure, which comes from `apps.list`. This is used when creating docker containers using `./run.sh -o .. -b 2`, which produces containers with `{APP_GROUPNAME}_{USERNAME}`. You can check these container names with `docker ps -a`. This APP_GROUPNAME is provided via docker_prefix so the scarab launch script can detect your containers and run containers in them. And example of an app groupname would be `docker_traces`
```

Comment:
allbench instead of docker_traces here again after this documentation is moved to README.LitzLab.md